doc/jenkins-conda.rst

```diff
@@ -0,0 +1,168 @@
+                """
+                }
+            }
+        }
+        stage("Start builder"){
+            steps {
+                script {
+                    sh """
+                    docker run --name ${container_name} -di --rm \
+                        --env TS_CONFIG_{name_of_config}_DIR=/home/saluser/ts_config_{name_of_config} \
+                        --env LSST_DDS_DOMAIN=citest \
+                        -v ${WORKSPACE}:/home/saluser/source ${dockerImageName}
+                    """
+                }
+            }
+        }
+        stage("Clone ts_config_{name_of_config}"){
+            steps {
+                script {
+                    sh """
+                    docker exec ${container_name} sh -c "git clone https://github.com/lsst-ts/ts_config_{name_of_config}.git"
+                    """
+                }
+            }
+        }
+        stage("Create Conda package") {
+            when {
+                buildingTag()
+            }
+            steps {
+                script {
+                    sh """
+                    docker exec ${container_name} sh -c 'cd ~/source/conda && source ~/miniconda3/bin/activate && source "\$OSPL_HOME/release.com" && conda build --prefix-length 100 .'
+                    """
+                }
+            }
+        }
+        stage("Create Conda Dev package") {
+            when {
+                not {
+                    buildingTag()
+                }
+            }
+            steps {
+                script {
+                    sh """
+                        docker exec ${container_name} sh -c 'cd ~/source/conda && source ~/miniconda3/bin/activate && source "\$OSPL_HOME/release.com" && conda build -c lsstts/label/dev --prefix-length 100 .'
+                    """
+                }
+            }
+        }
+
+        stage("Push Conda Release package") {
+            when {
+                buildingTag()
+            }
+            steps {
+                withCredentials([usernamePassword(credentialsId: 'CondaForge', passwordVariable: 'anaconda_pass', usernameVariable: 'anaconda_user')]) {
+                    script {
+                        sh """
+                        docker exec ${container_name} sh -c "source ~/miniconda3/bin/activate && \
+                            anaconda login --user ${anaconda_user} --password ${anaconda_pass} && \
+                            anaconda upload -u lsstts --force \
+                            ~/miniconda3/conda-bld/linux-64/ts-${package_name}*.tar.bz2"
+                        """
+                    }
+                }
+            }
+        }
+    }
+    post {
+        cleanup {
+            sh """
+            docker stop ${container_name}
+            """
+        }
+    }
+    }
+
+Just change the set name to whatever the package name is.
+Add and change the dependencies to match your package.
+
+.. code::
+
+    {% set data= load_setup_py_data() %}
+    {% set name= "ts-athexapod" %}
```

Comment:
it still says `ts-athexapod` here... maybe replace with something like `csc_name`?